go.mod

```diff
@@ -32,13 +23,10 @@ require (
 	github.com/pkg/errors v0.8.1
 	github.com/sclevine/spec v1.4.0
 	github.com/spf13/cobra v0.0.5
-	github.com/spf13/viper v1.6.1 // indirect
-	github.com/timakin/bodyclose v0.0.0-20190930140734-f7f2e9bca95e // indirect
-	github.com/tommy-muehle/go-mnd v1.1.1 // indirect
-	github.com/ultraware/whitespace v0.0.4 // indirect
-	github.com/uudashr/gocognit v1.0.1 // indirect
 	golang.org/x/crypto v0.0.0-20191206172530-e9b2fee46413
+	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	gopkg.in/yaml.v2 v2.2.7 // indirect
+	gotest.tools/v3 v3.0.2 // indirect
```

Comment:
What are these changes about?

Reply:
@jromero we did perform a `go mod tidy`, so maybe something got pulled in through that process